### ISSUE TITLE
Use DNS records in addition to the meta tag.

### ISSUE THREAD

**Issue by sorcix** [mstoiber/pgp.asc#13]
PGP fingerprints and the location of the public key can be published in DNS. This is standardized already and even supported by GPG.

https://0x80.io/activities/documents/pgp.html

Consider supporting this as well, as people won't use 10 different "standards".
[Relevant](https://xkcd.com/927/)

**Comment by mxstbr** [comment]
Yes, I know of that and it is on my Todo List to add it to the site. I was very busy the last two weeks, so I didn't get to do much, but I should be able to work on the site this weekend!
